cmd/todos_server/main.go

```diff
@@ -1,21 +1,25 @@
 package main
 
 import (
-	"github.com/go-chi/chi"
 	"log"
 	"net/http"
 
-	impl "github.com/anz-bank/syslgen-examples/todos-impl"
+	"github.com/go-chi/chi"
+
+	impl "github.com/anz-bank/syslgen-examples/pkg/todos-impl"
 
-	"github.com/anz-bank/syslgen-examples/todos"
+	"github.com/anz-bank/syslgen-examples/gen/todos"
 )
 
+const serverPort = ":8080"
```

Comment:
Specify the full address: `serverAddr`. It can still be `":8080"`. It just shouldn't be restricted to specifying the port. (For local dev, "localhost:8080" avoids the macOS security popup.)